### ISSUE TITLE
Creating new Google doc opens in the wrong account

### ISSUE THREAD

**Issue by dash4385** [PierBover/AlfredWorkflowGoogleApps#5]
I log into 2 Gmail accounts (Work and Person). My Work account is the default account in Chrome. All of the open commands will open the windows in the correct default account. However the new command creates the the new documents in my person account and not the default.

**Comment by PierBover** [comment]
The extension only opens a URL in a browser. The session in your browser is what will decide which account will be used.

Maybe you have logged in to Google Docs with your other account?

**Comment by dash4385** [comment]
I try logging out of both and flipping the order I logged in so that my personal one would be the default and then everything opened in that one. So seems like whatever if the default account in Chrome doesn't matter. Not sure if there is a way to handle this since there is no unique URL for accounts. For now I duplicated the flows and slightly modified the keyword on the second one and have it open to a different browser. Now my works account opens in the Chrome browser and my personal account opens to another. This way I don't have to have 2 logins in one browser.
